packages/discovery-provider/src/queries/get_comments.py

```diff
@@ -19,12 +19,14 @@
 COMMENT_REPLIES_LIMIT = 3
 
 
-def get_replies(session, parent_comment_id):
+def get_replies(session, parent_comment_id, offset=0, limit=COMMENT_REPLIES_LIMIT):
     replies = (
         session.query(Comment)
         .join(CommentThread, Comment.comment_id == CommentThread.comment_id)
         .filter(CommentThread.parent_comment_id == parent_comment_id)
-        .limit(COMMENT_REPLIES_LIMIT)
+        .order_by(desc(Comment.created_at))
```

Comment:
This shows most recent first right? I think the goal is to have the oldest comments/replies first (same with root level comments)

Reply:
ooh true good call